CONTRIBUTING.md

```diff
@@ -7,3 +7,7 @@ model need a quick design convo first so we can keep things high-quality. Just
 [open an issue](https://github.com/openworkflowdev/openworkflow/issues/new)
 describing what you'd like to do. [This is a great
 example](https://github.com/openworkflowdev/openworkflow/issues/16).
+
+The same should happen for documentation and business facing stuff, like e.g. websites (though those are in general 'core-team' stuff). So, as a general rule:
```

Comment:
```suggestion
If you are unsure whether a change would be accepted, feel free to ask in an
issue or check for labels like `help wanted`, `good first issue`, or `bug` on
the repo.

If you want to take an existing issue, leave a comment so it can be assigned
unless it is already in progress by another contributor.

For new functionality or anything that adds major behavior, start with a short
design conversation in an issue. Share the problem, your proposed approach, and
why it belongs in OpenWorkflow. The core team will help confirm direction before
any PRs start.

PRs that skip these steps may be closed.
```